docs/INSTALL.md

```diff
@@ -508,13 +531,13 @@ NDK version **26.1.10909125** and CMake version **3.22.1** are installed.
 If you are low on RAM, disk space or traffic there are ways to reduce system requirements:
 - exclude the `cpp` folder from indexing. If you do not make any work on the C++ code, this will greatly improve the start-up performance and the ram usage of Android Studio. Click on the `Project` tab on the left, find the `cpp` folder (should be next to the `java` folder), right click on it and select `Mark Directory as` -> `Excluded` (red folder icon). Then restart Android Studio.
 - in Android Studio enable "File > Power Save Mode";
+- disable thhe "Android NDK Support" plugin in "Settings -> Plugins" completely and use less the annoying Visual Studio Code or Qt Creator for editing C++ code instead;
```

Comment:
"thhe"
and please avoid subjective judging ("less annoying") in docs :)

Reply:
better use more neutral wording like
"* consider using less resource intensive IDEs to work with C++ code, e.g. Visual Studio Code or Qt Creator"

Reply:
I think that "crappy" is a more appropriate adjective in the context of IntelliJ/AndroidStudio.

Reply:
Updated.